### PR TITLE
Update 2022 roadmap with the latest deliveries

### PR DESCRIPTION
**Pull request by brunoocasali**

### CODE REVIEW

File: roadmap/2022.md [hunk 1]
```diff
@@ -186,18 +186,18 @@ Meilisearch releases:
...
 - [ ] Wrap up the Java SDK and release the new enhanced version with the addition of support to
-  - [ ] v0.28 Meilisearch version
+  - [x] v0.28 Meilisearch version
   - [ ] v0.29 Meilisearch version
   - [x] Add analytics setup to the SDK.
 
 New projects:
 - [ ] Implement packer.io to build Meilisearch images for the cloud providers.
 - [ ] Build the action plan to create the MVP of the new Laravel-Scout-Integration-Extended on our side.
 - [ ] "Docscraper 2.0":
```
Comment (brunoocasali) [inline roadmap/2022.md:198]:
```suggestion
- [x] "Docscraper 2.0":
```

File: roadmap/2022.md [hunk 2]
```diff
@@ -186,18 +186,18 @@ Meilisearch releases:
...
-  - [ ] Do the complete research around autocomplete.js and docsearchbar v2 support with our JavaScript client.
-  - [ ] Provide the action plan to add support for those libraries in case they are incompatible.
+  - [x] Do the complete research around autocomplete.js and docsearchbar v2 support with our JavaScript client.
+  - [x] Provide the action plan to add support for those libraries in case they are incompatible.
```
Comment (brunoocasali) [inline roadmap/2022.md:200]: @bidoubiwa, is that correct to say that you were able to finish these two items? Cause I think so!

File: roadmap/2022.md (hunk 1 above)
Comment (bidoubiwa) [reply roadmap/2022.md:198]: Not sure this should be checked

Comment (brunoocasali) [reply roadmap/2022.md:198]: I checked it because the only two items we added are done :)

**Review by alallema — APPROVED** [review]
LGTM! 🎉

File: roadmap/2022.md (hunk 1 above)
Comment (bidoubiwa) [reply roadmap/2022.md:198]: AH yes, but in fact, docs-scraper is a whole different subject. We should have named it `Autocomplete`

**Pull request merged**